### ISSUE TITLE
Volltextsuche

### ISSUE THREAD

**Issue by bmix** [jfreax/OpenLaw#7]
Es scheint nicht möglich Gesetze nach Inhalt zu durchsuchen. Die Suche hält sich nur an die Namen der Gesetzbücher. Eine solche "Machete" wäre aber durchaus brauchbar im Dschungel der Gesetze. Besonders Nicht-Juristen (wie ich) könnten davon profitieren.

**Comment by jfreax** [comment]
Volltextsuche ist auch schon länger mein Wunsch. Leider aber nicht ganz so einfach umzusetzen. Bzw. ich muss noch ein paar Vorüberlegungen treffen. Immerhin liegen die Gesetzestexte nicht komplett lokal auf dem Gerät sondern sind nur online Verfügbar. Dazu muss ich als das Backend und die API dazu erweitern.

Ist aber definitiv mit auf meiner Todo Liste.

**Comment by bmix** [comment]
Verstehe. Cachst Du die bereits gelesenen lokal? Dann Volltext eben nur bei den bereits vorhandenen. Ich glaube die meisten werden eh eine Vorauswahl treffen.

**Comment by jfreax** [comment]
Ja es wird lokal gecached. Dazu muss man aber sagen, dass jeder Paragraph einzeln heruntergeladen wird. Also auch wenn man ein Gesetzbuch schon mal "angelesen" hat, kann die lokale Suche immernoch nur genau die entsprechenden Paragraphen durchsuchen.

Eine Möglichkeit wäre, ganze Gesetzbücher optional lokal herunterzuladen. Ähnlich dem Favorisieren. Und diese kann man dann Volltext durchsuchen...

**Comment by bmix** [comment]
Ja, so würde ich das anbieten. Voller Download gewünschter Bücher und dann Volltext in denen und, zusätzlich, im Cache, falls das Sinn macht und leicht zu coden ist.
